src/Module/Settings/TwoFactor/Index.php

```diff
@@ -90,7 +91,9 @@ protected function post(array $request = [])
 					break;
 			}
 		} catch (\Exception $e) {
-			notice(DI::l10n()->t('Wrong Password'));
+			if (!($e instanceof FoundException)) {
+				notice(DI::l10n()->t($e->getMessage()));
+			}
 		}
```

Comment:
```suggestion
			notice(DI::l10n()->t($e->getMessage()));
		} catch (FoundException $e) {
			// Nothing to see here, please disperse
		}
```
I'm not sure about the order of the `catch` calls.